src/osd/osd_types.cc

```diff
@@ -3142,6 +3156,18 @@ void pg_stat_t::decode(ceph::buffer::list::const_iterator &bl)
     if (struct_v >= 29) {
       decode(log_dups_size, bl);
     }
+    if (struct_v >= 30) {
+      uint16_t osd_to_respond;
+      decode(osd_to_respond, bl);
+      scrub_sched_status.m_osd_to_respond = osd_to_respond;
+      uint8_t tmp8;
+      decode(tmp8, bl);
+      scrub_sched_status.m_ordinal_of_requested_replica = tmp8;
+      decode(tmp8, bl);
+      scrub_sched_status.m_num_to_reserve = tmp8;
+    } else {
+      scrub_sched_status.m_num_to_reserve = 0;
```

Comment:
I'm assuming it's fine leave the 2 other members as-they-are.

**EDIT**: actually all of them are 0-initialized. Is there a reason for handling `m_num_to_reserve` differently?

Reply:
Agree. I'll remove the 'else' part.